/**:
  ros__parameters:

    # number of input topics
    number_of_input_topics: 3

    topic_names: ["/applanix/lvx_client/gnss/fix", "/sbg/ros/nav_sat_fix", "/sbg/ros/ekf_nav_sat_fix"]

    # origin latitude
    origin_latitude: 41.11678470009106

    # origin longitude
    origin_longitude: 29.016272522002062

    # origin altitude
    origin_altitude: 114.27148911158406